Gopkg.lock

```diff
@@ -63,6 +63,18 @@
 
 [[projects]]
   branch = "master"
+  name = "github.com/jkeiser/iter"
+  packages = ["."]
+  revision = "67b94d6149c6b2efe0fde15d36a50a7692c2da17"
+
+[[projects]]
+  branch = "master"
+  name = "github.com/jochenvg/go-udev"
```

Comment:
Please make sure we force the constraint to the master branch in the `Gopkg.toml` for this repo because we import it on purpose and we want to stick the latest version.
About `"github.com/jkeiser/iter"`, I guess this is a dependency pulled by the new package we have imported. That's okay not to constraint this one because we are not using it directly.

Always make sure you run
```
dep ensure
dep prune
```
after you have modified `Gopkg.toml`